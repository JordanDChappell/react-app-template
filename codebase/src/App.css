html {
  font-family: 'Courier New', Courier, monospace;
  font-size: 18px;
}
body {
  height: 100vh;
}
header {
  margin: 0;
  display: grid;
  place-items: center;
}

.app {
  text-align: center;
}
.heading {
  font-size: 3rem;
  font-weight: bold;
}
