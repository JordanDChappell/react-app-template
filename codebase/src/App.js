import React from 'react';

import './App.css';

const App = () => (
  <div className="app">
    <header>
      <h1 className="heading">React App Template</h1>
      <p>
        Edit <code>src/App.js</code> and save to reload.
      </p>
    </header>
  </div>
);

export default App;
