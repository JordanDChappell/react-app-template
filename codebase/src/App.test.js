import React from 'react';
import { render, screen } from '@testing-library/react';
import renderer from 'react-test-renderer';

import App from './App';

describe('App', () => {
  it('Matches snapshot', () => {
    const component = renderer.create(<App />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
  it('Renders', () => {
    render(<App />);
    expect(screen.getByText('React App Template')).toBeInTheDocument();
  });
});
