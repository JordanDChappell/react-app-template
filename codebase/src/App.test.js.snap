// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Matches snapshot 1`] = `
<div
  className="app"
>
  <header>
    <h1
      className="heading"
    >
      React App Template
    </h1>
    <p>
      Edit 
      <code>
        src/App.js
      </code>
       and save to reload.
    </p>
  </header>
</div>
`;
